CIP-????/README.md

```diff
@@ -0,0 +1,530 @@
+amount of activity off-chain (like in the high frequency trading example) would likely more than
+make up for any remaining duplicate information in each transaction/block. Furthermore, the impact
+this CIP has on the average block size seems smaller than the impact the main alternative proposals
+would have on the average block size.
+
+## Alternatives
+
+### Validation Zones
+
+[Validation Zones](https://github.com/cardano-foundation/CIPs/pull/862) is another intent settlement
+system proposal, but it is unable to properly model all intents since it doesn't allow for
+unbalanced intents (ie, intents without counter-party information). As a result of the model not
+accurately representing all intents, the proposal requires a lot of extra complexity and friction
+with its usage.
+
+The Validation Zones CIP fundamentally models intents as full transactions which means all intents
+must be balanced. Without the ability to specify unbalanced intents, Validation Zones forces users
+to try to guess what actions the counter-party will use when satisfying their intent. In practice,
+this means Alice must use input and output **placeholders** in her intent, and these placeholders
+must also specify the value of the UTxO that will eventually go there; these placeholders are
+necessary to finish balancing the intent transaction. But this creates problems for potential
+counter-parties: 
+
+- Only UTxOs that exactly fit those placeholders can be used
+- The counter-party can't add or remove placeholders after the fact
+
+Imagine if Alice wants to swap 100 ADA for 50 DJED; this says nothing about how the counter-party
+will satisfy her swap. Perhaps Bob (the counter-party) wants to provide the DJED, but it is
+currently broken up across three UTxOs. And perhaps he wants the 100 ADA he receives to be stored in
+two separate outputs. If Alice specifies only one input placeholder and one output placeholder in
+her intent transaction, Bob might be unable to act as her counter-party. If Bob really wanted to
+still trade with Alice, he would have to find a way to merge his inputs and outputs into one input
+and one output. This may not always be possible and Bob may have to give up on being Alice's
+counter-party. In other words, because Alice was forced to specify placeholders in her intent, she
+missed out on a potential trading partner.
+
+DApps would also be unable to be direct liquidity sources for these intents due to not always being
+able to match the value specified in each placeholder. If the input place holder is for 10.3 ADA but
+the liquidity source UTxO actually has 20.5 ADA, this liquidity source cannot be used despite it
+having enough liquidity. The Validation Zones CIP argues that DApps can just add support for
+breaking-off/merging UTxOs so that the liquidity source UTxOs can be forced into the proper forms
+for the required placeholders. However, this would dramatically increase the complexity and attack
```

Comment:
I don't see where this is discussed in the Validation Zones CIP. But I also don't see a big problem here, just by going on with the same logic from the last paragraph. A matching transaction can always spend the 'problematic' outputs and then construct the remainder of the transaction as it would construct the transaction piece. I don't see where this impacts 'dramatically increases complexity'.

Reply:
> A matching transaction can always spend the 'problematic' outputs and then construct the remainder of the transaction as it would construct the transaction piece. I don't see where this impacts 'dramatically increases complexity'.

I guess I'm just biased. Given there is a path that doesn't require these extra steps, why wouldn't you take that path? Why would you ever take the path of *more* resistance?